Permanent/ViewControllers/MembersViewController.swift

```diff
@@ -63,7 +88,10 @@ class MembersViewController: BaseViewController<MembersViewModel> {
     }
     
     fileprivate func getMembers() {
+    
+        showSpinner()
```

Comment:
shouldnt we have an isBusy event in the viewmodel and based on that show the spinner or not?